src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    
+    <div *ngIf="childrendataSource.data.length > 0 && !isLoading" class="table-wrapper">
+      <table mat-table [dataSource]="childrendataSource"  >
+        <tr hidden="true">
+          <th scope="col" >Child Name </th>
+          <th scope="col"  >DateOfBirth</th>
+          <th scope="col" >AfterBirthdayAge</th>
+          
+        </tr>
+        <ng-container matColumnDef="name">
+          <td mat-cell *matCellDef="let child">{{ child.name}} 
```

Comment:
Instead of just displaying the `name` you can use the `<app-child-block>` component which shows the name, the project number and the picture of a child.
See the example below.